deviations/templates/deviations/_remove_modal.html

```diff
@@ -0,0 +1,64 @@
+{% load i18n %}
+{% load course %}
+
+<div class="modal" id="remove-selected-modal" tabindex="-1" role="dialog" aria-labelledby="remove-selected-modal-label">
+	<div class="modal-dialog" role="document">
+		<div class="modal-content">
+			<div class="modal-header">
+				<button type="button" class="close" data-dismiss="modal" aria-label="{% translate "CLOSE" %}">
+					<span aria-hidden="true">&times;</span>
+				</button>
+				<h4 class="modal-title" id="remove-selected-modal-label">
+					{% translate "REMOVE_SELECTED_DEVIATIONS" %}
+				</h4>
+			</div>
+
+			<div class="modal-body">
+				<p id="remove-selected-hidden-warning" class="alert alert-warning">
+					{% translate "SOME_DEVIATIONS_ARE_HIDDEN_WARNING" %}
+				</p>
+				<p id="remove-selected-no-selection-warning" class="alert alert-warning">
+					{% translate "NO_DEVIATIONS_SELECTED_WARNING" %}
+				</p>
+				<p id="remove-selected-confirmation"></p>
+			</div>
+
+			<div class="modal-footer">
+				<input
+					class="aplus-button--secondary aplus-button--md"
+					type="button"
+					value="{% translate 'CANCEL' %}"
+					data-dismiss="modal"
+				/>
+				<input
+					id="remove-selected-submit-button"
+					class="aplus-button--danger aplus-button--md"
+					type="submit"
+					value="{% translate 'REMOVE' %}"
+					formaction="{{ remove_url }}"
+				/>
```

Comment:
The reason I added `formaction` to the remove modal's submit button instead of adding `action` to the form, is that we might want to use the selection checkboxes for something other than deletion in the future. In that case we would likely add another modal with a submit button with a different `formaction`.

Reply:
Also note that the selected deviations are no longer removed with AJAX even though that was originally implemented in #401. I changed that because:
- The user can now delete multiple deviations at a time using the checkboxes, so the page reload is not a big deal anymore.
- The deviations are grouped in the backend. Reloading allows us to regroup the deviations when some of them are deleted.